extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -907,15 +907,26 @@ class MultipleBackendTask extends ReplicateObject {
     }
 
     _putObjectTagging(sourceEntry, log, cb) {
-        this.retry({
-            actionDesc: 'send object tagging XML data',
-            entry: sourceEntry,
-            logFields: { entry: sourceEntry.getLogInfo() },
-            actionFunc: done => this._putObjectTaggingOnce(
-                sourceEntry, log, done),
-            shouldRetryFunc: err => err.retryable,
-            log,
-        }, cb);
+        const dataStoreVersionId = sourceEntry.getReplicationSiteDataStoreVersionId(this.site);
+        async.series([
+            next => {
+                // put object if it wasn't already replicated.
+                // A replicated object has a dataStoreVersionId
+                if (!dataStoreVersionId) {
+                    return this._putObject(sourceEntry, log, next);
```

Comment:
If we replicate get/put the object at this stage, is there still a need to update the object's tag? 
I assume its metadata (tags...) is already up-to-date.

Reply:
you're right, i wasn't aware that putting the object also puts in the updated tags, fixed it